src/bin/restful-react-import.ts

```diff
@@ -38,7 +38,11 @@ program.option("--config [value]", "override flags by a config file");
 program.parse(process.argv);
 
 const createSuccessMessage = (backend?: string) =>
-  chalk.green(`${backend} 🎉  Your OpenAPI spec has been converted into ready to use restful-react components!`);
+  chalk.green(
+    `${
+      backend ? `${backend} ` : ``
```

Comment:
Thanks so much! Let's make this a little more concise! 

```suggestion
      backend || ""
```